lib/mix/lib/mix/tasks/format.ex

```diff
@@ -561,7 +563,7 @@ defmodule Mix.Tasks.Format do
 
   defp format_file({file, formatter}, task_opts) do
     input = read_file(file)
-    output = formatter.(input)
+    output = (bit_size(input) > 0 && formatter.(input)) || <<>>
```

Comment:
Do we still need this change, given the change to `elixir_format`?

Reply:
This should skip empty files when formatting, but the change to `elixir_format` skips formatting empty lines, and an empty file is just an empty line. Removed this piece of code from the PR as it was redundant. Thanks!